docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,20 @@
+# Akka Enhancements
```

Comment:
Akka Diagnostics

Reply:
changed

Reply:
Now it is almost identical, except for the obvious names.